app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
+    if len(parts) != 3 or parts[0] != "cp":
+        raise ValueError("Error")
+
+    com, f_name, s_name = parts
+
+    if f_name != s_name:
+        with open(f_name, "r") as first, open(s_name, "w") as second:
```

Comment:
Consider using more descriptive variable names for file handles, such as `source_file` and `destination_file`, to improve code readability as suggested in the checklist.